js/index.js

```diff
@@ -74,30 +74,28 @@ function displayProducts(data) {
     data.docs.forEach(element => {
         htmlRepresentation += `
         <div class="item">
-            <div class="class="col-sm-12 col-lg-6">
+            <div class="card-group" style="height: 69vh">
                 <div class="card h-100" style="width: 18rem;">
                     <img src=${element.bild_app} class="card-img-top"  alt="Card image cap">
-                    <div class="card-body">
+                    <div class="card-body h-100">
                     <h5 class="card-title">${element.titel}</h5>
                     <p class="card-text">${element.beschreibung}</p>
                     </div>
                     <ul class="list-group list-group-flush">
                     <li class="list-group-item font-weight-bold">Price: <span class="text-danger h4">${element.preis} Euro</span></spand></li>
-                    <li class="list-group-item list-price text-muted" id="basic-price">Basic Price: <span class="text-danger">${element.basicPrice}</span></spand></li>
+                    <li class="list-group-item text-muted ${element.basicPrice ? "yesList" : "noList" }">Basic Price: <span class="text-danger">${element.basicPrice}</span></spand></li>
```

Comment:
here you can apply only one class to only hide element and do nothing if it's fine to be displayed.